migrations/2020/Version20201128164940.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20201128164940 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Allow more recipients for payments';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        $this->addSql(<<<'SQL'
+CREATE TABLE pa_payment_email_recipients (
```

Comment:
*Nitpick*

Proč to není normálně odsazený? :grin:

Reply:
ok